File: python/7 kyu/217.py
# https://www.codewars.com/kata/569924899aa8541eb200003f/train/python

# Passed

from string import ascii_uppercase

def quicksum(packet):
    alphabet_to_value_dict = {
        alphabet: value for value, alphabet in enumerate(ascii_uppercase, start=1)
    }
    alphabet_to_value_dict.update({" ": 0})
    
    result = 0
    for pos, char in enumerate(packet, start=1):
        if char not in alphabet_to_value_dict:
            result = 0
            return result
        
        result += alphabet_to_value_dict[char] * pos
        
    return result

output = quicksum("ACM")
print(output)

File: nodejs/7 kyu/217.js
// https://www.codewars.com/kata/5174a4c0f2769dd8b1000003/train/javascript

// Passed

function solution(nums) {
  let result = nums ? nums.sort((a, b) => a - b) : [];
  return result;
}

const output = solution([1, 2, 3, 10, 5]);
console.log(output);